### ISSUE TITLE
🛠️ [TASK] : Add Semantic identifier on onboarding elements

### ISSUE THREAD

**Issue by emiride** [input-output-hk/catalyst-voices#3202]
### Summary

Add Semantic identifier on onboarding elements

### Description

Semantic identifier are not visible by people who enable accessibility, but are part of the DOM and are a standard way of locating specific elements when Flutter is rendered on the Web.
